docs/manual/developer/03_creating_content.md

```diff
@@ -558,6 +558,61 @@ controls:
       and to use the targeted policy.
 ```
 
+### Reporting status
+
+In some cases, it's useful to know the status of a certain control for a
+specific product. In order to better portray this, it's possible to set
+such information on each control using the `status` key.
+
+The `status` key may hold the following values:
+
+* `pending`: The control is not yet evaluated for the product.
+
+* `not applicable`: The control is not applicable to this product.
+
+* `inherently met`: The control is inherently met by the product.
+
+* `documentation`: The control is addressed by product documentation.
+
+* `planned`: The control is not yet implemented, but is planned.
+
+* `partial`: While work has been done to address this control, there is still
+             work needed to fully address it.
+
+* `complete`: The control is addressed by the product.
```

Comment:
And `status: complete` seems equivalent to  `automated: yes`

Reply:
uhm... now that I think about it, we might want to distinguish between "the product can do this" and "the product can do this & we can already check this through content"

Reply:
perhaps we should have something like:
`status: implemented` to portray that the system implements it... and then...
`status: automated` to portray that we have automation in place to check it's compliance in the content.

Reply:
That makes sense to me, although `implemented` could be confused with "we have rules implemented".
How about `supported`, as in the product has features that can support the control.

Reply:
I dig that!